types/password/password-tests.ts

```diff
@@ -0,0 +1,3 @@
+import password from 'password';
```

Comment:
Did you mean `import password = require('password');`?  The module doesn't appear to have a default export.

Reply:
You're right. I've replaced with your code.